exercises/largest-series-product/largest_series_product_test.go

```diff
@@ -2,27 +2,39 @@ package lsproduct
 
 import "testing"
 
-const targetTestVersion = 3
+const targetTestVersion = 4
 
-func TestLargestSeriesProduct(t *testing.T) {
+func TestTestVersion(t *testing.T) {
 	if testVersion != targetTestVersion {
 		t.Fatalf("Found testVersion = %v, want %v", testVersion, targetTestVersion)
 	}
+}
+
+func TestLargestSeriesProduct(t *testing.T) {
 	for _, test := range tests {
 		p, err := LargestSeriesProduct(test.digits, test.span)
-		switch {
-		case err != nil:
-			if test.ok {
+		if !test.ok { // expect error
+			// check if err is of error type
+			var _ error = err
```

Comment:
but then it wouldn't fit with "Ignore the normal return value when an error expected" of #282